tests_integration/nativeapp/test_version.py

```diff
@@ -372,12 +372,18 @@ def test_nativeapp_version_create_package_no_magic_comment(
 
         # app package contains version v1 with 2 patches
         actual = runner.invoke_with_connection_json(split(list_command))
-        for row in actual.json:
-            # Remove date field
-            row.pop("created_on", None)
-            # Remove metric_level field
-            row.pop("metric_level", None)
-        assert actual.json == snapshot
+        assert len(actual.json) == 2
+        _assert_list_of_dicts_contains_key_with_value(actual.json, "patch", 0)
```

Comment:
This is significantly weakening the assertion. Since the JSON output is part of the contract, I'd prefer checking that the actual dict is a superset of the expected one instead. Do you want to make the change or would you prefer that we do it?

Reply:
I didn't know which fields should I include, treat this as a proposition. Would be great if you could do this 🙏 .

Reply:
ok feel free to close this, I'll submit a PR soon.